app/views/courses/CoursesView.coffee

```diff
@@ -81,6 +85,13 @@ module.exports = class CoursesView extends RootView
     if hocCourseInstance
       @courseInstances.remove(hocCourseInstance)
 
+    _.forEach Object.keys(courseIDs), (courseID) =>
+      levels = new Levels()
+      @listenTo levels, 'sync', =>
+        @originalSlugMap[level.get('original')] = level.get('slug') for level in levels.models
+        @render?()
+      @supermodel.trackRequest(levels.fetchForCourse(courseID, { data: { project: 'original,slug' }}))
+
```

Comment:
Shouldn't this view be fetching levels for the classroom, not the course? Since the classroom has the local copy of the levels for the courses, which may be different.